openedx/core/lib/django_require/staticstorage.py

```diff
@@ -4,9 +4,10 @@
 
 from pipeline.storage import PipelineCachedStorage
 from require.storage import OptimizedFilesMixin
+from django_pipeline_forgiving.storages import PipelineForgivingStorage
 
 
-class OptimizedCachedRequireJsStorage(OptimizedFilesMixin, PipelineCachedStorage):
+class OptimizedCachedRequireJsStorage(OptimizedFilesMixin, PipelineForgivingStorage):
```

Comment:
In the forgiving model, where/how are errors logged? Are we going to know about missing assets?